src/scripts/notify.sh

```diff
@@ -144,6 +144,7 @@ ShouldPost() {
 # This is done so this script may be tested.
 ORB_TEST_ENV="bats-core"
 if [ "${0#*$ORB_TEST_ENV}" = "$0" ]; then
+    . $BASH_ENV
```

Comment:
Add `# shellcheck disable=SC1090` to the line above to ignore this error.
https://github.com/koalaman/shellcheck/wiki/SC1090

We are safe to ignore this, the variable is known within CircleCI.